Assets/Scripts/DayNightCycle/DayAndNightControl.cs

```diff
@@ -0,0 +1,247 @@
+﻿using Assets.Scripts.Services;
+using Assets.Scripts.Settings;
+using UnityEngine;
+using UnityEngine.UI;
+using System;
+using UnityEngine.Animations;
+
+namespace Assets.Scripts.DayNightCycle
+{
+    public class DayAndNightControl : MonoBehaviour
+    {
+        public GameObject moon;
+        public float sunTilt = -15f;
+        [SerializeField] private TimecycleProfile timecycle = null;
+        [SerializeField] private float sunRotationOffset = 0f;
+        [Tooltip("The amount of frames to wait before doing the next lighting update")]
+        [SerializeField] private int lightingUpdateInterval = 10;
+        public Material skyBoxPROCEDURAL;
+
+        [Range(0f, 24f)] public float currentTime;
+        public float CurrentTime01 => currentTime / 24;
+        public Slider TimeSlider  = null;
+        public Camera MoonCamera = null;
+        public Camera MainCamera = null;
+        public int currentDay = 0;
+        public Light directionalLight;
+        public float DayLength = 300f; //default value is 300 seconds in one day
+        public bool pause { get; set; }
+        
+        [HideInInspector]
+        float lightIntensity; //static variable to see what the current light's insensity is in the inspector
```

Comment:
@FransStreicher are you sure you're doing this one right? You want it to be visible in the editor but you used a private float and gave it the `HideInInspector` attribute on top of that.